app/controllers/forms_controller.rb

```diff
@@ -22,6 +23,12 @@ def show
   end
 
   def submit
+    if @current_user.present?
+      @current_user.update_attributes(
+        params.require(:user)
+        .permit(:first_name, :last_name, :email)
+      )
+    end
```

Comment:
I didn't have anything in mind. But do we have validations on these fields which these parameters could not meet? If so, we need to handle that.